ani.voskanyan/repo/Lesson_3/exercise_01/exercise_01.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
+#a. Using one statement with one print function 
+print(("*" * 19 + "\n") * 4)
+
+#b. Using four statement with one print function
+print(("*" * 19 + '\n') + ("*" * 19 + '\n')  + ("*" * 19 + '\n') + ("*" * 19 + '\n'))

```

Comment:
RW: Try without the "\n" newline character.

Reply:
Should the statement be in one line and without the "\n" character at the same time? Do I understand correctly?